inspirehep/modules/workflows/views/callback.py

```diff
@@ -104,7 +103,32 @@ def _find_and_continue_workflow(workflow_id, recid, result=None):
             'message': 'workflow with id %s not found.' % workflow_id,
         }
 
-    return {'status': 'succeeded'}
+    return {
+        'status': 'succeeded',
+        'message': 'workflow with id %s continued.' % workflow_id,
+    }
+
+
+def _put_workflow_in_error_state(workflow_id, error_message, result):
+    workflow_object = workflow_object_class.get(workflow_id)
+    if not workflow_object:
```

Comment:
I am not sure about this, if I try in the shell, if `get` fails an exception is thrown:

```
In [19]: workflow_object_class.get(12323)
---------------------------------------------------------------------------
WorkflowsMissingObject                    Traceback (most recent call last)
<ipython-input-19-45b961fd0548> in <module>()
----> 1 workflow_object_class.get(12323)

/opt/inspire/lib/python2.7/site-packages/invenio_workflows/api.pyc in get(cls, id_)
    154             except NoResultFound:
    155                 raise WorkflowsMissingObject("No object for for id {0}".format(
--> 156                     id_
    157                 ))
    158             return cls(model)

WorkflowsMissingObject: No object for for id 12323
```